src/qspdlog_model.hpp

```diff
@@ -26,6 +26,12 @@ class QSpdLogModel : public QAbstractListModel
     void setMaxEntries(std::optional<std::size_t> maxEntries);
     std::optional<std::size_t> getMaxEntries() const;
 
+    void setForegroundMapping(const std::string& loggerName, std::optional<QBrush> brush);
+    std::optional<QBrush> getForegroundMapping(const std::string& loggerName) const;
+
+    void setBackgroundMapping(const std::string& loggerName, std::optional<QBrush> brush);
+    std::optional<QBrush> getBackgroundMapping(const std::string& loggerName) const;
```

Comment:
`const std::string&` -> `std::string_view`
```suggestion
    void setForegroundMapping(std::string_view loggerName, std::optional<QBrush> brush);
    std::optional<QBrush> getForegroundMapping(std::string_view loggerName) const;

    void setBackgroundMapping(std::string_view loggerName, std::optional<QBrush> brush);
    std::optional<QBrush> getBackgroundMapping(std::string_view loggerName) const;
```